.github/workflows/publish.yml

```diff
@@ -9,6 +9,10 @@ on:
       localfs_ref:
         description: 'localfs package ref'
         required: false
+      forge_ui_components_release_name:
```

Comment:
forge_ui_components are now merged into the main forge app and shouldn't be a separate package now

Reply:
Removed dependency to this package in the pipeline.